src/sentry/static/sentry/app/views/organizationEventsV2/loadingContainer.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+
+import LoadingIndicator from 'app/components/loadingIndicator';
+
+export default function LoadingContainer(props) {
+  const {children, hasResults, isLoading} = props;
+
+  return (
+    <div>
+      {isLoading && (
+        <div>
```

Comment:
Does this need to be `position: relative`? Is the `div` needed at all?

Reply:
Oops, yeah. Meant to move this out of the parent container into here